go/vt/vtgate/vstream_manager.go

```diff
@@ -565,6 +567,11 @@ func (vs *vstream) sendAll(sgtid *binlogdatapb.ShardGtid, eventss [][]*binlogdat
 			}
 		}
 		vs.eventCh <- events
```

Comment:
This line should be dropped since we now send this in the `select` below

Reply:
good catch, sorry about that.